subiquity/models/installpath.py

```diff
@@ -38,26 +38,30 @@ def paths(self):
         if self.cmdline_sources:
             cmdline = [(_('Install from cli provided sources'), 'cmdline')]
         return cmdline + [
-            (_('Install Ubuntu'),                 'ubuntu'),
+            (_('Install Ubuntu'),                         'ubuntu'),
             (_('Install MAAS bare-metal cloud (region)'), 'maas_region'),
             (_('Install MAAS bare-metal cloud (rack)'),   'maas_rack'),
         ]
 
-    @property
-    def sources(self):
```

Comment:
This removed property is referenced by `get_seed_yaml` here https://github.com/mwhudson/subiquity/blob/refactor-config-creation/subiquity/ui/views/snaplist.py#L415

Not sure if we want to keep the sources property, or make the snaplist reference and transverse the full rendered config instead?

Currently it bobms out with assertionerror. So can't merge this.